test/check-sosreport

```diff
@@ -35,11 +34,7 @@ class TestSOS(MachineCase):
         b.click('[data-target="#sos"]')
         b.wait_visible("#sos")
 
-        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1278807
-        # Sosreport waits 5 minutes for a hung ausearch, so we wait 10 minutes
-        # for sosreport.
-        with b.wait_timeout(600):
```

Comment:
In any case, isn't the default (I believe 60 seconds) very low here? I would think something like 2 minutes should always be allowed, especially on possibly slower test machines.